### DISCUSSION TITLE
Map stream key to user

### DISCUSSION THREAD

**Discussion by domainzero** [OvenMediaLabs/OvenMediaEngine#1574] [Ideas]
It would be nice if you could map a stream key to a username and then access the stream via a username, thus obfuscating the stream key but still having the stream accessible via a nice name.

This allows a kind of "twitchlike" access, where someone has a private stream key that functions as a pass key, but their stream is accessible by a well known username.

**Comment by naanlizard** [comment]
Isn't that generally what your backend software should be doing? The app you put in front of ome?
